src/device_info.sh

```diff
@@ -218,6 +219,39 @@ function get_os()
   device_info_data['desktop_environment']="$desktop_env"
 }
 
+# This function populates the bootloader variables from the
+# device_info_data variable.
+#
+# @target Target machine
+function get_bootloader()
+{
+  local target="$1"
+  local flag="$2"
+  local ip="$3"
+  local port="$4"
+  local cmd='cat "$etc_path"/*-release | grep -w ID | cut -d = -f 2'
```

Comment:
This etc_path variable is not set, so running
`cat "$etc_path"/*-release | grep -w ID | cut -d = -f 2`
leads to
`cat: '/*-release': No such file or directory`

Maybe to get the distribution, you could expand `detect_distro()` in src/kwlib.sh and make it detect raspbian, since it apparently doesn't.